### DISCUSSION TITLE
Lsposed and afwall+

### DISCUSSION THREAD

**Discussion by MrHappy** [LSPosed/LSPosed#1514] [Q&A] [answered]
When I open the 'parasite'-shortcut it cannot uodate the module list. When I disable afwall it can, but there is no application logged.
Which app do I need to grant access to be able to uodate the module-list AND keep afwall active?

**Comment by giostark** [comment, marked as answer]
Hi,
It take me too a while understand it. 
The name of the item is "**shell**". 
If you don't enable this you are not able to update the list in LSPosed and keeping on AFWall at the same time.

**Reply by mocsa** [reply]
Be careful that if your phone is set to your local language (something other than English) then in AFWall the word "shell" will probably be translated into your local language. So in AFWall look for the word whatever "shell" means in your phone's current language.
